acceptance/uk/gov/hmrc/apisubscriptionfields/acceptance/ExternalServices.scala

```diff
@@ -14,7 +14,23 @@
  * limitations under the License.
  */
 
-package uk.gov.hmrc.apisubscriptionfields.util
+package uk.gov.hmrc.apisubscriptionfields.acceptance
+
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
```

Comment:
License is duplicated

Reply:
OK will fix this. No idea how this happened.